### ISSUE TITLE
`finally` is not a function

### ISSUE THREAD

**Issue by fergarrui** [kafkajs/confluent-schema-registry#54]
Hi,
I'm getting an issue only with 1.0.5 version:

```
Crash: KafkaJSNumberOfRetriesExceeded: this.api.Schema.find(...).finally is not a function","groupId":"<REDACTED>","retryCount":0,"stack":"KafkaJSNonRetriableError\n  Caused by: TypeError: this.api.Schema.find(...).finally is not a function\n    at SchemaRegistry.getSchemaOriginRequest (<REDACTED>/node_modules/@kafkajs/confluent-schema-registry/src/SchemaRegistry.ts:163:2)\n    at SchemaRegistry.getSchema
<...>
```
The version 1.0.4 does not contain that `finally` and works.

Surprisingly, I don't see other issues created at the repo.

Do you have any ideas?

Thanks!

**Comment by Nevon** [comment]
My guess would be that you have some non-native Promise implementation that is being used, which doesn't have the `finally` method available: https://repl.it/@TommyBrunn/PersonalImperturbableMass

Can you share a bit about your environment? Are you using TS/JS? Where is the code running?

**Comment by fergarrui** [comment]
Hey, thanks for your answer.
I'm using TS, yep, using NestJS and running the code locally or in docker both fail for me.

Thanks!

**Comment by Nevon** [comment]
What version of Typescript and NodeJS are you using? `Promise.finally` is available from TS 2.7 and Node 10.

**Comment by fergarrui** [comment]
Ok, I was not using node10, my bad, sorry for wasting your time.

Thanks!
